tensorflow/java/src/test/java/org/tensorflow/TensorTest.java

```diff
@@ -105,55 +116,90 @@ public void createFromBufferWithNonNativeByteOrder() {
   }
 
   @Test
-  public void createWithTypedBuffer() {
-    int[] ints = {1, 2, 3, 4};
-    float[] floats = {1f, 2f, 3f, 4f};
-    double[] doubles = {1d, 2d, 3d, 4d};
-    long[] longs = {1L, 2L, 3L, 4L};
-    long[] shape = {4};
+  public void whenCreateUsingDoubleBufferThenExpectedMatrixIsStoredInTensor() {
```

Comment:
My mistake, saw matrix sprinkled throughout a lot of tests (for matrix data) and put the wrong term in there. Will fix that up.